package.json

```diff
@@ -11,7 +11,8 @@
     "coverage": "jest --coverage",
     "codecov": "npx codecov",
     "start": "start-storybook -p 8000 -c storybook",
-    "storybook": "build-storybook -c storybook -o docs/out/storybook"
+    "storybook": "build-storybook -c storybook -o docs/out/storybook",
+    "plop": "npm run plop"
```

Comment:
perhaps the script should have a more semantic name like `create-component` or something similar